### DISCUSSION TITLE
DONE: Please add a mouse icon similar to the circle for RR overlay, when Follow Mouse is turned on.

### DISCUSSION THREAD

**Discussion by I3igAl** [WowOpenBox/WowOpenBox#100] [Ideas]
Simple, would really like to see an overlay indicator for follow mouse, just like RR.

**Comment by mooreatv** [comment]
Implemented in 4.2.1
https://github.com/WowOpenBox/WowOpenBox/commit/1fcc09ae2a7d4a992d8132356e0378592f5b1310

**Comment by Quigle** [comment]
This indicator seems incorrect. When i move to my other screens, the icon remains on my main screen.
Please also allow turning this feature off (and/or moving the icon's location or changing its size), as I don't like it in such a prominent location of my screen.

**Reply by mooreatv** [reply]
> This indicator seems incorrect. When i move to my other screens, the icon remains on my main screen.

It's a mode indicator like the RR one it's not telling you where your mouse is (your mouse cursor does that), why should that indicator move?

> Please also allow turning this feature off (and/or moving the icon's location or changing its size), as I don't like it in such a prominent location of my screen.

Click "Overlay Config"

**Reply by mooreatv** [reply]
![image](https://user-images.githubusercontent.com/46745059/112740267-6a53a180-8f30-11eb-9f26-db5d92787e3d.png)

**Reply by Quigle** [reply]
Much appreciated. (I was looking at all the toggles in "Options" menu.)
